src/addon/cc-addon-option.js

```diff
@@ -0,0 +1,139 @@
+import '../atoms/cc-img.js';
+import '../atoms/cc-toggle.js';
+import { assetUrl } from '../lib/asset-url.js';
+import { css, html, LitElement } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { dispatchCustomEvent } from '../lib/events.js';
+import { i18n } from '../lib/i18n.js';
+
+/**
+ * A component that allows to enable or disable an add-on option.
+ *
+ * * 🎨 default CSS display: `block`
+ * <br>
+ * 🧐 [component's source code on GitHub](https://github.com/CleverCloud/clever-components/blob/master/src/addon/cc-addon-option.js)
+ *
+ * ## Details
+ *
+ * * If `enabled` is set to true, the option will be enabled by default. Otherwise, it will be disabled.
+ * * Emits an `input` event when the option is enabled or disabled.
```

Comment:
I'm not sure we need those details. The `enabled` is already documented in properties and the `cc-addon-option:input` is also documented in events.